api.planx.uk/modules/send/idox/nexus.ts

```diff
@@ -84,14 +84,24 @@ export async function sendToIdoxNexus(
 
   try {
     // Request 1/4 - Authenticate
-    const { token, organisation, organisationId } =
+    const { token, organisations, authorities } =
       await authenticate(uniformClient);
 
+    // Temporary guard clause for testing phase
+    //   Ensures payload `files` will always be api.editor.planx.dev links which Idox has token to download (unlike api.XXX.planx.pizza)
+    if (process.env.NODE_ENV !== "staging") {
+      return res.status(200).send({
+        message: `Successfully authenticated to Idox Nexus, skipping submission because not staging env`,
+        organisations: organisations,
+        authorities: authorities,
+      });
+    }
```

Comment:
This is an interesting point to discuss: 
- When we submit the ODP Schema to partners to test, it's their responsibility to download the associated `files` uploaded by the user
  - These are S3 downloads via our API - we issue each supplier staging & prod keys, but these don't cover `api.XXXX.planx.pizza` links
- Do we want to be able to test more easily from pizzas? Should we adjust staging API keys to cover both envs?? Or are we happy with test payloads exclusively coming from staging & prod?

Reply:
Update from dev call: this was my confusion! Our file API keys are scoped _per AWS environment_, not _per S3 bucket_ - so because both the pizza user data & staging user data buckets live on the same environment, the staging file API key issued will be able to download files from both/either :+1: